ReactWindows/ReactNative/Views/WebView/ReactWebViewManager.cs

```diff
@@ -0,0 +1,289 @@
+﻿using Newtonsoft.Json.Linq;
+using ReactNative.UIManager;
+using ReactNative.UIManager.Annotations;
+using System;
+using System.Collections.Generic;
+using Windows.UI.Xaml.Controls;
+using ReactNative.Views.WebView.Events;
+using Windows.Web.Http;
+
+namespace ReactNative.Views.WebView
```

Comment:
I'd be okay with changing the folder and namespace to Web so you don't have to use the fully qualified name for the WebView element below.